lib/common/tests/xml/crm_xml_init_test.c

```diff
@@ -98,7 +98,7 @@ create_attr_node(void **state) {
     xmlDocPtr doc = xmlNewDoc((pcmkXmlStr) "1.0");
     xmlNodePtr node = xmlNewDocNode(doc, NULL, (pcmkXmlStr) "test", NULL);
     xmlAttrPtr attr = xmlNewProp(node, (pcmkXmlStr) PCMK_XA_NAME,
-                                 (pcmkXmlStr) "value");
+                                 (pcmkXmlStr) PCMK_XA_VALUE);
```

Comment:
This is not an attribute name but a literal string used as a dummy value for the "name" attribute. You can rename it something like "test-attr" if you want to avoid confusion :)

Reply:
Thanks